src/helix-ui/styles/components/drawer.less

```diff
@@ -0,0 +1,54 @@
+@import (reference) 'vars';
+
+hx-drawer:not([hx-defined]) {
+  display: none; // prevent FOUC
+}
+
+// These styles should not be shared with ShadyDOM
+hx-drawer:extend(.hxComponent) {
+  background-color: @gray-100;
+  display: block;
+  height: 0;
+  padding: 0;
+  position: fixed;
+  width: 0;
```

Comment:
Height and width are set to 0 to prevent the background color from showing in the `<hx-drawer>` element.